mytest.json

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
This file seems to be checked in by mistake.

Reply:
Haha yep! I thought I'd deleted the output, but it must have slipped by.